dev/acf-export.json

```
@@ -2337,6 +2337,44 @@
                 "width": "",
                 "height": ""
             },
+            {
+                "key": "field_5d543d9c27f4c",
+                "label": "Degree Video Caption",
+                "name": "degree_video_caption",
+                "type": "text",
+                "instructions": "Text displayed under the video on degrees",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "default_value": "",
+                "placeholder": "",
+                "prepend": "",
+                "append": "",
+                "maxlength": ""
+            },
+            {
+                "key": "field_5d543dde69294",
+                "label": "Degree Spotlight",
+                "name": "degree_spotlight",
+                "type": "wysiwyg",
+                "instructions": "The content displayed under the video on degrees.",
```

Same as above, this could be updated to something more instruction-like:
```suggestion
                "instructions": "Enter spotlight content that will be displayed under the video for this degree.",
```